.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug Report
+about: Create a report to help us solve bugs!
+labels: bug
+---
+
+<!--
+        ✰  Thanks for opening an issue! ✰
+  Before submitting please review the template.
+  Please also ensure that this is not a duplicate issue :)
+-->
+
+## Summary
+
+<!-- Concisely describe the issue -->
+
+## Steps to Reproduce
+
+<!-- What commands in order should someone run to reproduce your problem? -->
+
+## Expected behavior
+
+<!-- A clear and concise description of what you expected to happen -->
+
+## Screenshots
+
+<!-- If applicable, add screenshots to help explain your problem. -->
+
+## Version
+
+- OS Name: [e.g. Ubuntu]
+- OS Version [e.g. 20.04]
+- CLI Version (output of `topos --version`)
+
```

Comment:
suggest adding a placeholder for any additional environmental or version details. In my experience, people tend to leave it out unless requested (and even then it is no guarantee, but more frequently provided than if not requested)

Reply:
Oups actually I went too fast here and left this irrelevant "topos".

Reply:
Additional Context is probably okay, but the intent of my comment was really about additional environmental and version details. Are you are open to modifying the additional context as with something like this?

<!-- Add any other context and environmental details about the problem here -->